.idea/codeStyles/Project.xml

```diff
@@ -14,15 +14,9 @@
       <option name="SPACES_WITHIN_OBJECT_TYPE_BRACES" value="false" />
       <option name="IMPORT_SORT_MODULE_NAME" value="true" />
     </JSCodeStyleSettings>
-    <JetCodeStyleSettings>
-      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
-    </JetCodeStyleSettings>
```

Comment:
Removed all code styles of languages that aren't directly used in the project. I think we could also remove the JS/TS stuff because of prettier, but I think that might affect the editor's recommended behavior.

I think IDEA has an option to match Prettier configured style, but I haven't played around too much with that yet